libiocage/lib/JailConfig.py

```diff
@@ -293,20 +292,19 @@ def _set_jail_zfs_dataset(self, value, **kwargs):
         self.data["jail_zfs_dataset"] = " ".join(value)
 
     def _get_jail_zfs(self):
-        enabled = self.data["jail_zfs"] == "on"
+        enabled = libiocage.lib.helpers.parse_bool(self.data["jail_zfs"])
         if not enabled:
             if len(self.jail_zfs_dataset) > 0:
                 raise libiocage.lib.errors.JailConigZFSIsNotAllowed(
-                    logger=self.logger
-                )
+                    logger=self.logger)
         return enabled
 
     def _set_jail_zfs(self, value, **kwargs):
         if (value is None) or (value == ""):
             del self.data["jail_zfs"]
             return
```

Comment:
there's a bunch of places in this file where we use `stringify()` and i'm not sure if we need to look into that, too